flink-formats/flink-hadoop-bulk/src/test/java/org/apache/flink/formats/hadoop/bulk/HadoopPathBasedPartFileWriterTest.java

```diff
@@ -82,13 +83,13 @@ public void testWriteFile() throws Exception {
         DataStream<String> stream =
                 env.addSource(new FiniteTestSource<>(data), TypeInformation.of(String.class));
         Configuration configuration = new Configuration();
-
+        // Use day format , in case of this test case run across two hours
```

Comment:
That's a more reasonable comment - but I would argue that the fix is still not good enough in that case. What if the test runs at midnight? We would still have the same issue?

Reply:
I got your message , let me think a better fix .

Reply:
@XComp  a new fix